monkestation/code/modules/mob/living/simple_animal/friendly/dog.dm

```diff
@@ -19,3 +19,42 @@
 	gold_core_spawnable = NO_SPAWN
 	unique_pet = TRUE
 	speak = list("barks!", "woofs!", "borks!", "yips!", "burks!")
+
+/mob/living/simple_animal/pet/dog/corgi/borgi
+	name = "E-N"
+	real_name = "E-N"
+	desc = "It's a borgi."
+	icon = 'monkestation/icons/mob/pets.dmi'
+	icon_state = "borgi"
+	icon_living = "borgi"
+	icon_dead = "borgi_dead"
+	var/emagged = FALSE
+	minbodytemp = 0
+	loot = list(/obj/effect/decal/cleanable/robot_debris)
+	del_on_death = TRUE
+	deathmessage = "blows apart!"
+	animal_species = /mob/living/simple_animal/pet/dog/corgi/borgi
+	nofur = TRUE
+	var/next_beep
+	var/beepsound = 'sound/items/timer.ogg'
```

Comment:
"beepsound" doesn't need to exist.
You can just put the file straight into the playsound itself.